roles/sap_ha_pacemaker_cluster/tasks/configure_nwas_ascs_ers_postinstallation.yml

```diff
@@ -192,20 +263,37 @@
       ansible.builtin.shell: |
         /usr/sap/hostctrl/exe/sapcontrol -nr {{ sap_ha_pacemaker_cluster_nwas_abap_ers_instance_nr }} -function HACheckConfig
       changed_when: false
+      failed_when:
+        - "'ERROR' in __sap_ha_pacemaker_cluster_register_where_ers.stdout"
 
-    - name: "SAP HA Pacemaker - (SAP HA Interface) Display HA check results for ASCS"
+    - name: "SAP HA Pacemaker - (SAP HA Interface) Get HA config for ASCS"
       when:
         - __sap_ha_pacemaker_cluster_register_where_ascs.rc == 0
+      become: true
+      become_user: "{{ sap_ha_pacemaker_cluster_nwas_abap_sid | lower }}adm"
+      register: __sap_ha_pacemaker_cluster_register_ascs_ha_config
+      ansible.builtin.shell: |
+        /usr/sap/hostctrl/exe/sapcontrol -nr {{ sap_ha_pacemaker_cluster_nwas_abap_ascs_instance_nr }} -function HAGetFailoverConfig
+      changed_when: false
```

Comment:
This `false` is obsolete?

Reply:
Strange that it was removed. I will add it again and retest.